ts/features/pn/aar/store/selectors/index.ts

```diff
@@ -28,6 +28,14 @@ export const aarAdresseeDenominationSelector = (
     case sendAARFlowStates.fetchingNotificationData:
     case sendAARFlowStates.displayingNotificationData:
     case sendAARFlowStates.notAddresseeFinal:
+    case sendAARFlowStates.notAddressee:
+    case sendAARFlowStates.creatingMandate:
```

Comment:
What about inverting this switch behavior? We map the cases there there's no `iun` and no `recipientInfo` with `return undefined` and move the `if (...) return denomination` to the default case